dashboard/app/controllers/regional_partners_controller.rb

```diff
@@ -119,4 +145,14 @@ def regional_partner_params
   def restricted_users
     User.select(RESTRICTED_USER_ATTRIBUTES_FOR_VIEW)
   end
+
+  def sanitize_date_hash(date_hash)
```

Comment:
I don't see this being called anywhere, so I'm having a hard time picturing what the `date_hash` will look like  - could you give an example? (Even just as a comment on the PR)

Reply:
It's leftover from the old way of doing minimum dates